docs/os/windows/windows-tools41.md

```diff
@@ -0,0 +1,292 @@
+| Qubes Video Driver                   |        +       |         -         |
+| Qubes Network Setup                  |        +       |         +         |
+| Private Volume Setup (move profiles) |        +       |         +         |
+| File sender/receiver                 |        +       |         +         |
+| Clipboard Copy/Paste                 |        +       |         +         |
+| Application shortcuts                |        +       |         +         |
+| Copy/Edit in Disposable VM           |        +       |         +         |
+| Block device                         |        +       |         +         |
+| USB device                           |        +       |         +         |
+| Audio                                |        +       |         +         |
+
+Qubes Windows Tools are open source and are distributed under a GPL license.
+
+> **Notes:**
+> - Currently only 64-bit versions of Windows 7, 10 and 11 are supported by Qubes Windows Tools. Only emulated SVGA GPU is supported (although [there has been reports](https://groups.google.com/forum/#!topic/qubes-users/cmPRMOkxkdA) on working GPU passthrough).
+> - This page documents the process of installing Qubes Windows Tools in version **R4.1**.
+
+Preparation
+-----------
+
+Before proceeding with the installation we need to disable the Windows mechanism that allows only signed drivers to be installed, because currently the drivers we provide as part of the Windows Tools are not digitally signed with a publicly recognizable certificate. To do that:
+
+ 1. Start the command prompt as Administrator, i.e. right click on the Command Prompt icon (All Programs -> Accessories) and choose "Run as administrator"
+ 2. In the command prompt type `bcdedit /set testsigning on`
+ 3. Reboot your Windows VM
+
+In the future this step will not be necessary anymore, because we will sign our drivers with a publicly verifiable certificate. However, it should be noted that even now, the fact that those drivers are not digitally signed, this doesn't affect security of the Windows VM in 'any' way. This is because the actual installation `iso` file can be verified as described in step 3 below. The only downside of those drivers not being signed is the inconvenience to the user that he or she must disable the signature enforcement policy before installing the tools.
+
+> **Note:** it is recommended to increase the default value of Windows VM's `qrexec_timeout` property from 60 (seconds) to, for example, 300. During one of the first reboots after Windows Tools installation Windows user profiles are moved onto the private VM's virtual disk (private.img) and this operation can take some time. Moving profiles is performed in an early boot phase when `qrexec` is not yet running, so timeout may occur with the default value. To change the property use this command in `dom0`: *(where `<VMname>` is the name of your Windows VM)*
+
+		[user@dom0 ~] $ qvm-prefs <VMname> qrexec_timeout 300
```

Comment:
That might be not enough in case applying significant updates (Windows 10 release upgrades) specially with old hardware (hdd). Qvm-create-windows-qube set 999999, my proposal in [PR](https://github.com/QubesOS/qubes-core-admin/pull/441) is 6000

Reply:
O.k. - with some comment on that.